spec/javascripts/member-ui/fundraiser_spec.js

```diff
@@ -260,7 +260,7 @@ describe("Fundraiser", function() {
       }
 
       beforeEach(function(){
-        suite.followUpUrl = "/pages/636/follow-up";
+        suite.followUpUrl = `/api/member_authentication/new?page_id=1&email=`;
```

Comment:
These files are not transpiled, so you can't use ES6. Unfortunately, when the js test runner can't parse a file, it just drops it on the floor. If you look on CircleCI, you'll see that it's only running 52 tests, but it should really be running 115 tests. it's just skipping this entire file.